contribution/generator/src/TestGenerator.php

```diff
@@ -53,7 +53,11 @@ public function createTestsFor(
             $method = $this->builderFactory->method($methodName)
                 ->makePublic()
                 ->setReturnType('void')
-                ->setDocComment("/**\n * uuid: {$case->uuid}\n * @testdox {$description}\n */")
+                ->setDocComment("/**\n * uuid: {$case->uuid}\n */")
+                ->addAttribute(new Node\Attribute(
+                    new Node\Name(\PHPUnit\Framework\Attributes\TestDox::class),
+                    [new Node\Arg(new Node\Scalar\String_($description))],
```

Comment:
```suggestion
                    [new Node\Arg(new Node\Scalar\String_(ucfirst($description)))],
```

In case we assume PHP8.4 for using the generator: `mb_ucfirst()`.

Reply:
This is already done on line 48: https://github.com/homersimpsons/exercism-php/blob/33ebb3bc26f00df67921e5708d519a11752a2ca3/contribution/generator/src/TestGenerator.php#L48